tests/drivers/display/display_read_write/src/main.c

```diff
@@ -14,8 +14,14 @@ LOG_MODULE_DECLARE(display_api, CONFIG_DISPLAY_LOG_LEVEL);
 static const struct device *dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_display));
 static const uint32_t display_width = DT_PROP(DT_CHOSEN(zephyr_display), width);
 static const uint32_t display_height = DT_PROP(DT_CHOSEN(zephyr_display), height);
+#if defined(CONFIG_RENESAS_RA_GLCDC)
```

Comment:
Instead of making this specific to the GLCDC, why not add KConfigs for this sample to set the section name and alignment of the display buffer? Then these can be setup in a board overlay for the Renesas boards

Reply:
I fixed it